### PR TITLE
bug fix: Add simple proptest model, fix a couple bugs in expected page size.

### PR DESCRIPTION
**Pull request by spacejam**
**Choose one:** is this a 🐛 bug fix

 Add simple proptest model, fix a couple bugs in expected page size.

## Checklist

- [x] tests pass
- [x] tests and/or benchmarks are included

## Semver Changes

up to you

### CODE REVIEW

File: tests/model.rs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+fn model(bit: usize, page_sz: usize) {
+  let mut bits = Bitfield::new(page_sz);
+  assert_eq!(bits.set(bit, true), Change::Changed);
+  assert_eq!(bits.get(bit), true);
+}
+
+proptest! {
+  #[test]
+  fn doesnt_crash(bit in 0usize..1_000_000, page_sz_exponent in 0usize..30) {
```
Comment (ralphtheninja) [inline tests/model.rs:15]: Sorry for newb question. But how does this work? The parameters in a function have a range and proptest uses this to invoke it for all pairs somehow?

Comment (spacejam) [reply tests/model.rs:15]: It assigns randomly chosen values in the specified ranges to `bit` and `page_sz_exponent`, sort of like a fuzzer with fine-grained control over specific things to randomize.

Comment (ralphtheninja) [reply tests/model.rs:15]: Ok cool. But is it the `proptest!` macro doing this or this is a feature of rust itself?

Comment (spacejam) [reply tests/model.rs:15]: proptest randomly chooses a value within a `Range`. in rust, `n..m` is shorthand for an instance of the `std::ops::Range` type.

**Review by yoshuawuyts — APPROVED** [review]
real neat; thanks!

**Comment by yoshuawuyts** [comment]
`v0.3.3` :tada:

**Pull request merged**